widgets/common/src/DynamicDropdown.jsx

```diff
@@ -218,7 +214,8 @@ DynamicDropdown.defaultProps = {
     prefetch: false,
     multiple: false,
     className: '',
-    refreshEvent: null
+    refreshEvent: null,
+    itemsFormatter: options => options
```

Comment:
Having it named `options` may be misleading, as it's not options, but the items from the request. How about

```suggestion
    itemsFormatter: items => items
```

Reply:
My bad, fixed.